library/ts-rollup/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
     "outDir": "./build/lib",
-    "target": "ESNext",
-    "module": "ESNext",
+    "target": "ES2018",
```

Comment:
⚠️ Need to modify the **lib** & **target** for the **typescript** plugin in **rollup.config.js**